spec/models/refinery/jobs/job_application_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'spec_helper'
+require 'helper/generator_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.